src/views/users/User/UserList.vue

```diff
@@ -129,6 +130,15 @@ export default {
           ]
         },
         columnsMeta: {
+          name: {
+            formatter: DetailFormatter,
+            formatterArgs: {
+              route: 'UserDetail',
+              routeQuery: {
+                tab: 'Basic'
+              }
+            }
+          },
           mfa_level: {
             width: '130px',
             formatter: (row) => {
```

Comment:
The code is missing a file path after `Vue.createInstance`, such as `"./App.vue"`. Additionally, there seems to be some syntax errors including an incorrect closing parenthesis on line 49 of the second component's function signature. These issues require minor adjustments; they don't seem to cause significant problems but will help improve readability and maintainability.